Source/CarthageKit/FrameworkExtensions.swift

```diff
@@ -275,11 +275,11 @@ extension URL {
 			throw error(failureReason: "Expected single resource value: «actual count: \(values.count)».")
 		}
 
-		guard case (keys.first!, let volumeSupportsFileCloning as CFBoolean)? = values.first else {
-			throw error(failureReason: "Unable to extract a CFBoolean from «\(String(describing: values.first))».")
+		guard let volumeSupportsFileCloning = values[key] as? NSNumber else {
```

Comment:
The document says the return value is `NSNumber`: https://developer.apple.com/documentation/foundation/urlresourcekey/1642798-volumesupportsfilecloningkey